src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -15,11 +15,13 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Displays all persons in the address book as a list with index numbers.\n"
             + "Example: " + COMMAND_WORD;
+    
+    private static final boolean IS_FOUND = true;
 
 
     @Override
     public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        return new CommandResult(getMessageForPersonListShownSummary(allPersons, IS_FOUND), allPersons);
```

Comment:
Consider overloading the method `getMessageForPersonListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed)` to call `getMessageForPersonListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed, boolean isFound)` to abstract away the private static final boolean IS_FOUND from the ListCommand class.